webapp/channels/src/components/sidebar/sidebar_header/sidebar_header.tsx

```diff
@@ -124,30 +121,27 @@ const SidebarHeader = (props: Props) => {
             <SidebarHeaderContainer
                 id={'sidebar-header-container'}
             >
-                <OverlayTrigger
-
-                    delayShow={Constants.OVERLAY_TIME_DELAY}
-                    placement='bottom'
-                    overlay={currentTeam.description?.length ? (
-                        <Tooltip id='team-name__tooltip'>{currentTeam.description}</Tooltip>
-                    ) : <></>}
+                <MenuWrapper
+                    onToggle={handleMenuToggle}
+                    className='SidebarHeaderMenuWrapper test-team-header'
                 >
-                    <MenuWrapper
-                        onToggle={handleMenuToggle}
-                        className='SidebarHeaderMenuWrapper test-team-header'
+                    <WithTooltip
+                        id='team-name__tooltip'
+                        title={currentTeam.description ? currentTeam.description : currentTeam.display_name}
+                        placement='bottom'
                     >
                         <SidebarHeading>
                             <button className='style--none sidebar-header'>
                                 <span className='title'>{currentTeam.display_name}</span>
                                 <i className='icon icon-chevron-down'/>
                             </button>
                         </SidebarHeading>
-                        <MainMenu
-                            id='sidebarDropdownMenu'
-                            usageDeltaTeams={usageDeltas.teams.active}
-                        />
-                    </MenuWrapper>
-                </OverlayTrigger>
+                    </WithTooltip>
+                    <MainMenu
+                        id='sidebarDropdownMenu'
+                        usageDeltaTeams={usageDeltas.teams.active}
+                    />
+                </MenuWrapper>
```

Comment:
WithTooltips wraps MenuWrapper as well, can you cross check previous code?

Reply:
Yes I noticed that, but the issue was even before the migrate the tooltip would not show when MenuWrapper was included. @M-ZubairAhmed , so I wasn't sure whether to wrap it or not.

Reply:
Oh do you mean with MainMenu under the withTooltip the tooltip doesnt work?

Reply:
Yeah @M-ZubairAhmed , so MenuWrapper requires exactly 2 children according to the error I was getting, that means I can't put MainMenu in the WithTooltip, it just returns a blank page. 
If I wrap the MenuWrapper in the WithTooltip (like in the initial implementation), the tootltip just doesn't show up. This was also the case when it was using the OverlayTrigger.

Reply:
Ah i see also the functionality is not affected here so i think approach is fine with me